### PR TITLE
olligranlund Iris + satan layouts

### PR DESCRIPTION
**Pull request by OlliGranlund**
My Iris + Satan layouts. Using Nordic layout so hopefully other Nordic people will find these usefull.

### CODE REVIEW

File: keyboards/iris/keymaps/olligranlund_nordic/keymap.c [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+enum custom_keycodes {
+	QWERTY = SAFE_RANGE,
+	LOWER,
+	RAISE,
+	ADJUST
+};
+
+#define KC_ KC_TRNS
+#define _______ KC_TRNS
```
Comment (drashna) [inline keyboards/iris/keymaps/olligranlund_nordic/keymap.c]: This line isn't necessary, as it's in the core keycodes, and can be removed.

File: keyboards/iris/keymaps/olligranlund_nordic/rules.mk [hunk 2]
```diff
@@ -0,0 +1,7 @@
+RGBLIGHT_ENABLE = yes
+BACKLIGHT_ENABLE = yes
+#EXTRAKEY_ENABLE = yes
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/iris/keymaps/olligranlund_nordic/rules.mk]: Could you remove this (ifndef/include/endif) block? 

It's not needed, and is a holdover from the old build system.

File: keyboards/satan/keymaps/olligranlund_ISO/config.h [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#include "../../config.h"
```
Comment (drashna) [inline keyboards/satan/keymaps/olligranlund_ISO/config.h]: This line is not actually needed, and should be removed. 

Specifically, the build script automatically includes all of the config.h files, so this is not necessary. 

Additionally, could you add `#pragma once` at the top of the file?

File: keyboards/satan/keymaps/olligranlund_ISO/keymap.c [hunk 4]
```diff
@@ -0,0 +1,112 @@
+#include "satan.h"
```
Comment (drashna) [inline keyboards/satan/keymaps/olligranlund_ISO/keymap.c]: Could you replace this include with `#include QMK_KEYBOARD_H` instead?

File: keyboards/satan/keymaps/olligranlund_ISO/keymap.c [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+#define BASE    0    // Default layer
+#define MOUSE   1    // Space layer
+#define NUMPAD  2    // Alt layer
+#define CAPS    3    // Caps layer
+
+#define MAC0 M(0) //
```
Comment (drashna) [inline keyboards/satan/keymaps/olligranlund_ISO/keymap.c]: This aren't being used.  You may want to remove them.

File: keyboards/satan/keymaps/olligranlund_ISO/rules.mk [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+NKRO_ENABLE = yes           # Nkey Rollover - if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+
+ifndef QUANTUM_DIR
```
Comment (drashna) [inline keyboards/satan/keymaps/olligranlund_ISO/rules.mk]: Could you remove this (ifndef/include/endif) block? 

It's not needed, and is a holdover from the old build system.

**Comment by drashna** [comment]
Could you make sure the satan keymap uses lower case characters. 

You may need to rename them to something else, commit, rename them back (but as lower case) and commit again to do that.

**Comment by OlliGranlund** [comment]
Damn, you are right. At first I just renamed it via my IDE but it apparently didn't make the change... git mv seemed to work tho.

**Comment by drashna** [comment]
Welcome to case sensitivity hell. :)
That's part of what I was talking about.  And some OSes (*cough* windows *cough*) handle it worse than others. 

But glad to see you got it renamed

**Comment by drashna** [comment]
Thanks!

**Comment by OlliGranlund** [comment]
No problem, the pleassure is mine :)

Will probably need to update the Iris keymap at some point, but that's a another day.

**Pull request merged**